doc/source/whatsnew/v0.19.0.txt

```diff
@@ -332,6 +332,7 @@ API changes
 - ``PeriodIndex.fillna`` with ``Period`` has different freq now coerces to ``object`` dtype (:issue:`13664`)
 - More informative exceptions are passed through the csv parser. The exception type would now be the original exception type instead of ``CParserError``. (:issue:`13652`)
 - ``astype()`` will now accept a dict of column name to data types mapping as the ``dtype`` argument. (:issue:`12086`)
+- ``Timestamp``, ``Period``, ``DatetimeIndex``, ``PeriodIndex`` and ``.dt`` accessor has ``.is_leap_year`` property to return logical whether date belong to leap year. (:issue:`13727`)
```

Comment:
Hmm, github is not really made for such linguistic review, would be much easier if I could just directly edit it .. :-)